packages/babel-preset-env/test/fixtures/sanity/regex-unicode-property/exec.js

```diff
@@ -0,0 +1 @@
+expect(/^\p{Unified_Ideograph}+$/u.test('中')).toBe(true);
```

Comment:
Could you check if `/s` flags work with unicode property escape?
```js
expect(/\p{Unified_Ideograph}./su.test('中\n')).toBe(true);
```

Reply:
It got SyntaxError `Expected atom ^\p{Unified_I` because `transform-dotall-regex` runs before `proposal-unicode-property-regex`.

However, if I moved `proposal-unicode-property-regex` before `transform-dotall-regex`, it would get a wrong result. see https://github.com/mathiasbynens/regexpu/issues/68

Reply:
When `proposal-unicode-property-regex` is applied _before_ `transform-dotall-regex`, the `.` token is transformed to equivalent code points given the default `dotAll: false`, rendered the subsequent `transform-dotall-regex` inactive.

https://github.com/babel/babel/blob/eb3767d58b5e376fa4f4e95e1ba9dc86c667908d/packages/babel-plugin-proposal-unicode-property-regex/src/index.js#L22-L25

Preferably `regexpu-core` could offer a `preserveDot: boolean` option so that `.` token can be consumed by `transform-dotall-regex` _after_ unicode-property transforms.

Note that we may workaround this by plug in `dotAllFlags` in `proposal-unicode-property-regex`,

```js
rewritePattern(node.pattern, node.flags, {
  dotAllFlags: node.flags.includes("s")
}
```
/cc @mathiasbynens